packages/astro/src/assets/services/service.ts

```diff
@@ -213,6 +213,10 @@ export const baseService: Omit<LocalImageService, 'transform'> = {
 			options.format = DEFAULT_OUTPUT_FORMAT;
 		}
 
+		if (!options.transform) {
```

Comment:
i thought of writing `options ??= 'eager'` but preferred to stick with `option.format` fashion